style.css

```diff
@@ -0,0 +1,48 @@
+* {
+  box-sizing: border-box;
+  margin: 0px;
+  padding: 0px;
+  border: 0px;
+}
+
+body {
+  background-color: #CCC;
+  font-family: Georgia, serif;
+  padding: 0px;
+  position: relative;
```

Comment:
If you don't have any child element with position: absolute this line doesn't do anything.